--- origsrc/tinycdb-0.76/Makefile	2006-06-28 10:17:37.000000000 -0500
+++ src/tinycdb-0.76/Makefile	2007-10-02 00:48:13.484375000 -0500
@@ -29,8 +29,8 @@
 LIBBASE = libcdb
 LIB = $(LIBBASE).a
 PICLIB = $(LIBBASE)_pic.a
-SHAREDLIB = $(LIBBASE).so.1
-SOLIB = $(LIBBASE).so
+SHAREDLIB = cygcdb-1.dll
+SOLIB = $(LIBBASE).dll.a
 CDB_USELIB = $(LIB)
 NSS_USELIB = $(PICLIB)
 LIBMAP = $(LIBBASE).map
@@ -38,9 +38,9 @@
 
 # The following assumes GNU CC/LD -
 # used for building shared libraries only
-CFLAGS_PIC = -fPIC
-CFLAGS_SHARED = -shared
-CFLAGS_SONAME = -Wl,--soname=
+CFLAGS_PIC =
+CFLAGS_SHARED = -shared -Wl,--enable-auto-image-base
+CFLAGS_SONAME = -Wl,--out-implib,
 CFLAGS_VSCRIPT = -Wl,--version-script=
 
 CP = cp
@@ -83,9 +83,8 @@
 
 $(SHAREDLIB): $(LIB_OBJS_PIC) $(LIBMAP)
 	-rm -f $(SOLIB)
-	ln -s $@ $(SOLIB)
 	$(CC) $(CFLAGS) $(CFLAGS_SHARED) -o $@ \
-	 $(CFLAGS_SONAME)$(SHAREDLIB) $(CFLAGS_VSCRIPT)$(LIBMAP) \
+	 $(CFLAGS_SONAME)$(SOLIB) $(CFLAGS_VSCRIPT)$(LIBMAP) \
 	 $(LIB_OBJS_PIC)
 
 cdb: cdb.o $(CDB_USELIB)
